### PR TITLE
Capitalize dictionary entries. Implements rightMove test and implemen…

### PR DESCRIPTION
**Pull request by rrustia**
…tation.

### CODE REVIEW

File: server/chess-opening-data/dictionary.js [hunk 1]
```diff
@@ -1,20 +1,19 @@
-export const Dictionary = {
-    phildor:
+const Dictionary = {
```
Comment (visaudi) [inline server/chess-opening-data/dictionary.js]: Camelcase please unless you were using a class.

File: server/rightmove.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+const Dictionary = require('./chess-opening-data/dictionary');
+
+const rightMove = (evalData) => {
```
Comment (visaudi) [inline server/rightmove.js]: What does evalData signify to other programmers?

File: server/rightmove.test.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
+const chai = require('chai');
+const expect = chai.expect;
+const assert = chai.assert;
+const should = chai.should;
+const { rightMove } = require('./rightmove');
+
+describe('function: rightMove testing', () => {
+
+    const evalData = {
```
Comment (visaudi) [inline server/rightmove.test.js]: Something more descriptive of the content.

File: server/chess-opening-data/dictionary.js [hunk 4]
```diff
@@ -12,9 +12,14 @@ export const Dictionary = {
                 { origin: "g1", destination: "f3" },
             black:   
                 { origin: "d7", destination: "d6" } 
+        },
+        {
```
Comment (rrustia) [inline server/chess-opening-data/dictionary.js]: Adds null value to let client know there's no more moves for that index.

File: server/.nyc_output/7fe66fc535e3fd182ec5d40c816b22b0.json [hunk 5]
```diff
@@ -0,0 +1 @@
...37}},"3":{"start":{"line":5,"column":22},"end":{"line":5,"column":41}},"4":{"start":{"line":6,"column":4},"end":{"line":6,"column":42}},"5":{"start":{"line":9,"column":0},"end":{"line":9,"column":31}}},"fnMap":{"0":{"name":"(anonymous_0)","decl":{"start":{"line":3,"column":18},"end":{"line":3,"column":19}},"loc":{"start":{"line":3,"column":33},"end":{"line":7,"column":1}},"line":3}},"branchMap":{},"s":{"0":1,"1":1,"2":2,"3":2,"4":2,"5":1},"f":{"0":2},"b":{},"_coverageSchema":"332fd63041d2c1bcb487cc26dd0d5f7d97098a6c","hash":"2146dfd79c1fc81f45f1e5c78e3e951ed82a8cd3","contentHash":"4e35f016ac4063fea05b0e2cce386e11_11.4.1"},"/Users/macbook/think/javascript/projects/chess-app/server/chess-opening-data/dictionary.js":{"path":"/Users/macbook/think/javascript/projects/chess-app/server/chess-opening-data/dictionary.js","statementMap":{"0":{"start":{"line":1,"column":19},"end":{"line":23,"column":1}},"1":{"start":{"line":25,"column":0},"end":{"line":25,"column":29}}},"fnMap":{},"branchMap":{},"s":{"0":1,"1":1},"f":{},"b":{},"_coverageSchema":"332fd63041d2c1bcb487cc26dd0d5f7d97098a6c","hash":"be1624c653880e4fa2159875b621e0f230fe41f0","contentHash":"dabf39368b0da7ba6fbb8b32678f4838_11.4.1"}}
```
Comment (visaudi) [inline server/.nyc_output/7fe66fc535e3fd182ec5d40c816b22b0.json]: Remove.

File: server/rightmove.test.js [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+    const moveAtIndex = { 
+        white : 
+            { origin: "e2", destination: "e4" },
+        black:   
+            { origin: "e7", destination: "e5" }
+    }
+
+    it('should return an object with the correct move pair for specific opening index', () => { 
+        console.log(rightMove(moveQuery) == moveAtIndex);
+        expect(JSON.stringify(rightMove(moveQuery))).to.equal(JSON.stringify(moveAtIndex));
```
Comment (visaudi) [inline server/rightmove.test.js]: Strings in variables.

**Pull request merged**